dipy/reconst/csdeconv.py

```diff
@@ -815,7 +818,7 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7,
     else:
         ci, cj, ck = roi_center
     w = roi_radius
-    roi = data[int(ci - w): int(ci + w), int(cj - w): int(cj + w), int(ck - w): int(ck + w)]
+    roi = data[int(ci - w): int(ci + w), int(cj - w)               : int(cj + w), int(ck - w): int(ck + w)]
```

Comment:
Did you mean to a line-break here? I'd put it after a comma, rather than in the middle of a slice... 

Also, while you're on this line: we should probably eliminate these spaces after colons.